packages/server/src/config/default.ts

```diff
@@ -5,6 +5,9 @@ const config: AppConfig = {
     redisPort: process.env.REDIS_PORT
         ? parseInt(process.env.REDIS_PORT, 10)
         : 6379,
+    redisDataCache: {
```

Comment:
not being used atm, but will add functionality to disable cache in tests in the next pr.